### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wdanilo**
### Pull Request Description

This PR updates the README so it looks nicer. I have a strange problem with prettier - the CI fails, while my prettier says everything is OK. I'd be thankful if someone could pretty-print this file, so we could merge it fast.

image

### CODE REVIEW

**Comment by iamrecursion** [comment]
You likely have the wrong version of prettier installed. Make sure you run `npm install` in the enso directory before `npx prettier --write .`, otherwise you're likely to be using the wrong version.

File: README.md [hunk 1]
```diff
@@ -46,67 +44,116 @@
...
-- 🌎 **Runs everywhere**  
-  Enso is available on MacOS, Windows, and Linux, and the Enso IDE runs on
-  web-native technologies. In time, you'll be able to run it in the web-browser,
-  giving even your tablet of phone access to your data.<br/><br/>
-
-<a href="https://www.youtube.com/watch?v=XReCQMZUmuE">See it in action.<br>
-
-<img alt="An example Enso graph" src="https://user-images.githubusercontent.com/1623053/105841783-7c1ed400-5fd5-11eb-8493-7c6a629a84b7.png">
-</a>
+<img align="left" width="36px" src="https://github.com/google/material-design-icons/blob/master/src/content/link/materialiconsround/24px.svg">
```
Comment (iamrecursion) [inline README.md:56]: All of these icons are almost impossible to see on dark-mode GH.

File: README.md [hunk 2]
```diff
@@ -21,9 +21,7 @@
   <br/>
 </p>
 
-### Get insights you can act on, fast
-
-<p>
+<p align="center">
```
Comment (iamrecursion) [inline README.md:24]: I can't comment on it, but the "Enso Language" header above isn't real text. Could we make it standard GH font?

Comment (wdanilo) [reply README.md:24]: We want to change it to something else (not contain the word "language" as well). This is not part of this PR change, so lets keep it as it for now, but I will create other PRs for improving README in the next days. This should be covered in some of these.

File: README.md (hunk 1 above)
Comment (wdanilo) [reply README.md:56]: We already have this issue with the logo on top of the page. The proper solution requires a lot of tricks (we cannot use predefined logos anymore, we need to fork them and change them). I'd propose to design README for now for the white theme and when we will be happy with it (I'm definitely not yet) then we will make it working in the dark mode as well. The reason is also that I'm not even sure if we will keep all the icons, as we will add here some graphics done by our website graphic designer, so lets first finish the README to an amazing state and then lets add dark omde support! :)

**Review by iamrecursion — APPROVED** [review]

**Pull request merged**